projects/saturn/src/main/java/io/fairspace/saturn/services/maintenance/MaintenanceService.java

```diff
@@ -22,7 +27,7 @@
 @Log4j2
 public class MaintenanceService {
     public static final String SERVICE_NOT_AVAILABLE = "Service not available";
-    public static final String REINDEXING_IS_ALREADY_IN_PROGRESS = "Reindexing is already in progress.";
+    public static final String REINDEXING_IS_ALREADY_IN_PROGRESS = "Maintenance is already in progress.";
```

Comment:
I would have two separate messages to distinct in logs which process is ongoing. Then aprove